content/bd/0004-relational-model-conversion.md

```diff
@@ -228,7 +226,7 @@ Tomemos dois exemplos, um com chave de um atributo e outro com chave de dois atr
 
 > product(<u>p_code</u>, p_name, price, stock)
 
-> order(<u>p_code, client_id</u>, quantity, date)
+> order(<u>p_code</u>, <u>client_id</u>, quantity, date)
```

Comment:
Hm, I think I prefer it to be all underlined, to give the meaning of a "composed key".
In the slides they use both interchangeably , all underlined and each attribute underlined, so I think both are correct.

@RafDevX thoughts?

Reply:
I'm quite sure @jjasferreira is right.

I get where you're coming from and what you're trying to convey, but each attribute should be underlined individually.
You can explain that intuition in text if you'd like.
What you're referring to in the slides happens once or twice and they're most likely typos.

See Wikipedia: https://en.wikipedia.org/wiki/Relational_model#Examples